CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to cybersoc.co.uk
+
+Thanks for wanting to contribute to our repo! Here are some guidelines we ask you to follow:
+
+## Issues
+
+If you find any issues with the site, please open an [issue](https://github.com/CyberSocYork/cybersocyork.github.io/issues/new/choose). This will allow us to fix the issue as soon as possible. Please be as descriptive as possible in your issue, as this will help us to reproduce the problem.
+
+## Changes to static pages
+
+If there is a change you would like to see made to anything on the website, feel free to open an issue requesting this.
+
+## Blog posts
+
+Our preferred method of accepting blog posts is via pull request, as this allows us to have the git commit history accurately reflect the authors of posts. If you do not feel comfortable submitting a pull request, we also accept submissions by contacting committee directly and submitting the `.md` file.
+
+### Commit guidelines
+
+- Please give commits descriptive messages.
+  - For example, if a commit adds a blog post titled `Why rust is the future of programming` , a suitable commit message would be `Added blog post "Why rust is the future of programming"`, not `Created file src/markdown-posts/rust-future.md`.
```

Comment:
Personal preference on this is to write commits in the imperative (e.g. "add images to file") but if you'd really prefer it in past tense then we can roll with it

Reply:
Personally I prefer past tense, e.g. 
"This commit fixed the error ..."

That's also not really the point of the guideline. The point is to stop people from just using the github default "added file" messages from using the web interface.

Reply:
Yep that's fair enough and a good idea tbh